docs/reference/base-specification.md

```diff
@@ -0,0 +1,126 @@
+
+## Implementation
+
+This is the current implementation of the recommended base technical specification above. The full operational definition of the process for generating the "base file" is:
+
+```
+# base: A version of the ontology that does not include any externally imported axioms.
+$(ONT)-base.owl:
+	robot merge -i $(SRC) -i $(OTHER_SRC) -i $(ALL_IMPORTS) \
+	reason --reasoner ELK --equivalent-classes-allowed asserted-only --exclude-tautologies structural \
+	relax \
+	reduce -r ELK \
+	remove --base-iri $(URIBASE)/OBA --axioms external --preserve-structure false --trim false \
+    {STANDARD_COMMANDS}
+	--output $@
+```
+
+_This is a breakdown of the sub-processes:_
+
+- `robot merge -i $(SRC) -i $(OTHER_SRC) -i $(ALL_IMPORTS)`:  Merge the ontology with all its dependencies together.
+- `reason --reasoner {REASONER} --equivalent-classes-allowed asserted-only --exclude-tautologies structural`: Add all inferred `rdfs:subClassOf` axioms.
+- `relax`: break down all `owl:equivalentClass` axioms into `rdfs:subClassOf` axioms where the subject is a "base entity"
+- `reduce -r {REASONER}`: remove all redundant `rdfs:subClassOf` axioms
+- `remove --base-iri {BASEIRI} --axioms external --preserve-structure false --trim false`: remove all axioms where the subject is not a "base entity"
+
+## How to determine the "subject" of an axiom
+
+Instead of exhaustively defining the idea of subject (extensional definition), we define it by inclusion (intentional definition):
+
+Given:
+
+- `?x` a base entity
+- `?y` an arbitrary expression (like a class or property name, or class expression)
+- `?property` an instance of `owl:AnnotationProperty`, `owl:ObjectProperty` or `owl:DataProperty`
+- `?annotation_property` an instance of `owl:AnnotationProperty`
+- `?object_property` an instance of `owl:ObjectProperty`
+- `?data_property` an instance of `owl:DataProperty`
+
+The following axioms are considered to having `?x` as a subject (aka "base axioms"):
+
+- Assertions of the form `?x  ?property ?value` (includes Annotation, Object and DataProperty Assertions)
+- SubClassOf axioms of the form `?x  rdfs:subClassOf ?y`
+- Equivalent class axioms of the form `?x  owl:equivalentClass ?y`
+- Equivalent class axioms of the form `?y  owl:equivalentClass ?x`
+- An assertion of a characteristic of the form `Characteristic(?x)`, like `TransitiveCharacteristic(?x)` etc.
+- An assertion of the form `ObjectPropertyDomain(?x, ?y)`
+- An assertion of the form `ObjectPropertyRange(?x, ?y)`
+
+Examples of axioms that _do not have `?x` as a subject_:
+
+- SubClassOf axioms of the form `?y rdfs:subClassOf ?x`
+- An assertion of the form `ObjectPropertyDomain(?y, ?x)` (where `?x` is the domain of an object property `?y`)
+- An assertion of the form `ObjectPropertyRange(?y, ?x)` (where `?x` is the range of an object property `?y`)
+- A GCI axiom of the form `SomeValuesFrom(?object_property, ?x) rdfs:subClassOf ?y`
```

Comment:
This is a bit of a question mark. What do you think? If this is not included, it will _never_ be included, unless the GCI is in the same ontology as the `?object_property`..

Reply:
@cmungall tends towards including this, but we need a better review here and think about it more.